test/quantization/fx/test_quantize_fx.py

```diff
@@ -7224,9 +7268,16 @@ def __init__(self):
                     self.emb = torch.nn.EmbeddingBag(num_embeddings=10, embedding_dim=12, mode='sum')
                     self.linear = torch.nn.Linear(12, 1).to(dtype=torch.float)
 
-                def forward(self, input: torch.Tensor, offsets: Optional[torch.Tensor] = None,
-                            per_sample_weights: Optional[torch.Tensor] = None):
-                    x = self.emb(input, offsets, per_sample_weights)
+                # Note: this does not work since the result is not scriptable
+                # because of these additional optional args
+                # def forward(self, input: torch.Tensor, offsets: Optional[torch.Tensor] = None,
```

Comment:
Hi @supriyar, do you know how EmbeddingBag is used currently? do we pass in all optional arguments right now? and is scriptability still a requirement?